app/jobs/order_import_job.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class OrderImportJob < ApplicationJob
+  def perform(order_import, report_recipient)
```

Comment:
nit: should we name this `perform!` so it's obvious it could raise an exception?

Reply:
this is defining active job entry point method which is inherited